docs/guide/mappings.rst

```diff
@@ -84,12 +86,91 @@ In Python
     UBERON:0000955 EFO:0000302 oio:hasDbXref
     ...
 
+Directionality of mappings
+--------------------------
+
+While mappings are often thought of as bidirectional, this is not quite true in practice:
+
+- Some mapping predicates such as ``skos:broadMatch`` and ``skos:narrowMatch`` are inherently directed, with the meaining inverted in the opposite direction
+- Many mappings are only be asserted in one ontology, and may be "invisible" when queried from the mapped ontology
```

Comment:
Might want to expand on why mappings may be "invisible" from an ontology, e.g. each ontology must agree to the mapping and include the mapping and actively assert the mapping to the term in the other ontology for it to be visible from either ontology.